code/game/mecha/mecha_defense.dm

```diff
@@ -113,7 +113,7 @@
 
 
 /obj/mecha/bullet_act(obj/item/projectile/Proj) //wrapper
-	if (!enclosed && occupant && !silicon_pilot) //allows bullets to hit the pilot of open-canopy mechs
+	if (!enclosed && occupant && !silicon_pilot && !Proj.force_hit && (Proj.def_zone == BODY_ZONE_HEAD || Proj.def_zone == BODY_ZONE_CHEST)) //allows bullets to hit the pilot of open-canopy mechs
```

Comment:
This is pretty obscure, might want to add some description to the mecha that tells what to aim at to hit the pilot.

Reply:
Would it be viable to add an examine text that only shows up when the examiner is holding a firearm of some sort?

Reply:
I've now got a line in the examine block of mecha.dm that adds an extra line to the examine text if the examiner is holding a ranged weapon (specifically, anything of /obj/item/gun);

"It looks like you can hit the pilot directly if you target the center or above."

Hopefully, this implies that a different event happens if you _don't_ target center or above. Thoughts?

Reply:
We can't fix stupid so this will be fine lol